go.work

```diff
@@ -0,0 +1,8 @@
+go 1.21.0
+
+use (
+	.
+	./pkg/util/xorm
+)
+
+replace xorm.io/xorm => ./pkg/util/xorm
```

Comment:
Can we comment on why we have this replace? (Basically following the other thread on the package name). Since this is first entry in this file, but also a bit of edge case, I think it is good to make that clear to people looking at this file.